ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -12,8 +12,9 @@ import utils 1.0
 ActivityNotificationBase {
     id: root
 
-    readonly property bool isOutgoingRequest: notification && notification.message.amISender
-    readonly property string contactId: notification ? isOutgoingRequest ? notification.chatId : notification.author : ""
+    readonly property bool isOutgoingMessage: notification && notification.message.amISender
+    readonly property string contactId: notification ? isOutgoingMessage ? notification.chatId : notification.author : ""
+    readonly property string contactName: contactDetails ? contactDetails.displayName || contactDetails.alias : ""
```

Comment:
I believe this should go thru `ProfileUtils.displayName()`, leaving out the nickname because you show that already as the `sender.secondaryName` below